share/wake/lib/system/remote_cache_api.wake

```diff
@@ -214,6 +218,14 @@ export def rscApiPostJob (req: CachePostRequest) (api: RemoteCacheApi): Result U
         | getCachePostRequestJson
         | formatJSON
 
+    def _ =
+        require True = shouldDebugRemoteCache Unit
+
+        def _ =
+            writeTempFile "remote.cache.api.postJob.req" "label: {req.getCachePostRequestLabel}\nreq: {jsonStr}"
```

Comment:
writeTempFile doesn't append to files, it just writes out a new file of shape `<prefix>.<content hash>` so unless the request is exactly identical each call here will create a new file.

Adding the new line makes it a little easier to see the different parts and since we don't have to worry about parsing out several requests from a file the single-line-per-request optimization doesn't give us much

Reply:
Over time, if we do have to do post processing on these files it would probably be good to make it more structured though. probably via JSON and by adding a timestamp

Reply:
Ah I was expecting these to be all put into one large log per prefix. Since thats not the case I agree that there is no benefit to not-line breaking